### PR TITLE
Add Feedback links to the bottom of every page

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

**Comment by evilnick** [comment]
image

This is a sample.

So, there are a few options:

-  Edit this page
-  Edit this page or file an issue (link to both)

File: pages/k8s/audit-logging.md [hunk 1]
```diff
@@ -158,6 +158,12 @@ Please refer to the upstream
 [Kubernetes Audit Webhook Backend documentation][k8s-audit-backend] for more
 information about the audit webhook config format and related options.
 
+<div class="p-notification--information">
+  <p class="p-notification__response">
+    Request edits to this page or file a bug <a href="https://github.com/charmed-kubernetes/kubernetes-docs/edit/master/pages/k8s/audit-logging.md" class="p-notification__action">here</a>
```
Comment (evilnick) [inline pages/k8s/audit-logging.md]: Maybe 'Submit edits to this page'?

Comment (tvansteenburgh) [reply pages/k8s/audit-logging.md]: How about, "Edit this page or file a bug here"?

Comment (evilnick) [reply pages/k8s/audit-logging.md]: ok  :+1:

**Comment by evilnick** [comment]
Two new variants to look at:


image

or if that is too stark, how about:


image

**Comment by evilnick** [comment]
@tvansteenburgh I prefer the second one if that helps

**Comment by tvansteenburgh** [comment]
> @tvansteenburgh I prefer the second one if that helps

Fine by me, thanks.

**Pull request merged**